openmp/runtime/cmake/config-ix.cmake

```diff
@@ -18,6 +18,7 @@ include(CheckIncludeFiles)
 include(CheckSymbolExists)
 include(LibompCheckFortranFlag)
 include(LLVMCheckCompilerLinkerFlag)
+include(OffloadOpenmpCommon)
```

Comment:
I still don't think this is necessary. Just check the version script independently in `offload/` and make OMPT support opt-in.